turbinia/evidence.py

```diff
@@ -164,20 +175,45 @@ class RawDisk(Evidence):
   """Evidence object for Disk based evidence.
 
   Attributes:
-    loopdevice_path: Path to the losetup device for this disk.
-    mount_path: The mount path for this disk (if any).
     mount_partition: The mount partition for this disk (if any).
+    path_to_disk: The path to the underlying data (block device or disk image).
     size:  The size of the disk in bytes.
   """
 
-  def __init__(self, mount_path=None, mount_partition=None, size=None, *args,
+  def __init__(self, mount_partition=None, size=None, *args,
                **kwargs):
     """Initialization for raw disk evidence object."""
-    self.loopdevice_path = None
-    self.mount_path = mount_path
+    super(RawDisk, self).__init__(*args, **kwargs)
     self.mount_partition = mount_partition
+    self.path_to_disk = self.local_path
     self.size = size
-    super(RawDisk, self).__init__(*args, **kwargs)
+    self._loopdevice_path = None
+    self._disk_mount_path = None
+
+  def copy_context(self, target_evidence):
```

Comment:
I guess this will get called by code in the other PRs then?

Reply:
Indeed